source/dub/project.d

```diff
@@ -1742,6 +1750,23 @@ final class SelectedVersions {
 		m_bare = false;
 	}
 
+	/** Constructs a new non-empty version selection, prefixing relative path
+		selections with the specified prefix.
+
+		To be used in cases where the "dub.selections.json" file isn't located
+		in the root package directory.
+	*/
+	this(Selected data, NativePath relPathPrefix)
+	{
+		this(data);
+		if (relPathPrefix.empty) return;
+		foreach (ref dep; m_selections.versions.byValue) {
+			const depPath = dep.path;
+			if (!depPath.empty && !depPath.absolute)
+				dep.path = relPathPrefix ~ depPath;
```

Comment:
```
source/dub/project.d(1766,5): Deprecation: function `dub.dependency.Dependency.path` is deprecated - Construct a new `Dependency` object instead
```

Reply:
Yeah thx, seen it via that awesome bot too, cool stuff! :+1: